src/main/rx/rx.c

```diff
@@ -73,6 +73,9 @@ const char rcChannelLetters[] = "AERT12345678abcdefgh";
 
 static uint16_t rssi = 0;                  // range: [0;1023]
 static uint8_t rssi_dbm = 130;             // range: [0;130] display 0 to -130
+#ifdef USE_RX_SNR_DBM
+static int8_t snr_dbm = -30;
```

Comment:
Magic number should be replaced with a const.

Reply:
will do.